articles/tokens/refresh-token.md

```diff
@@ -1,22 +1,25 @@
 ---
-description: A refresh token allows an application to request Auth0 to issue a new id_token directly, without needing to re-authenticate the user.
+title: Refresh Token
+description: What is a Refresh Token and what you can do with it.
 ---
 
-# Refresh Tokens
+# Refresh Token
 
-A **refresh token** is a special kind of [JWT](/jwt) that is used to authenticate a user without them needing to re-authenticate. This is primarily useful for mobile applications that are installed on a device.  
+A **refresh token** is a special kind of [JWT](/jwt) that contains the information required to obtain a new `access_token` or `id_token`.
```

Comment:
Refresh tokens are not necessarily JWTs. Auth0 might return them as JWTs, but client applications should have no expectation about their format.